### PR TITLE
1Me Corporate

### PR DESCRIPTION
**Pull request by MICHAEL-MAURICE**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/1Me Corporate/Readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+1Me is the easiest and fastest way to share your contact information.
+With 1Me, you can have an unlimited number of contact cards. You can choose which contact information to include on each card, and then choose who to share each card with.
+This is best for separating your business and personal contact information
+
+
+## Publisher: 1Me
+ 
+
+## Prerequisites
+You will need the following to proceed:
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/Readme.md:11]: @MICHAEL-MAURICE is it possible to provide links to the page where users can get the API key?

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/CardTempletes/": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:26]: @MICHAEL-MAURICE it is not allowed to have a default response only. please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). Please fix all responses in the swagger file.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:28]: @MICHAEL-MAURICE empty schema is not allowed. please define a proper schema. Please fix all schemas in the swagger file.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+          }
+        },
+        "operationId": "RetrieveCardTemplates",
+        "summary": "Retrieve all corporate card templates",
+        "description": "Retrieve all corporate card templates",
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:33]: @MICHAEL-MAURICE it is highly recommended to have a different description than the summary. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+            "description":"Disassociate members from your corporate",
+            "x-ms-summary":"Disassociate members from your corporate",
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json"
+          },
+          {
+
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:114]: @MICHAEL-MAURICE all files should be properly formatted. please remove all empty lines from the swagger file.

File: certified-connectors/1Me Corporate/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiProperties.json:18]: @MICHAEL-MAURICE it is not allowed to have the iconBrandColor set to "#007ee5". Icon must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

File: independent-publisher-connectors/The Meal DB/Readme.md [hunk 7]
```diff
@@ -1,37 +0,0 @@
-# TheMealDB
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/The Meal DB/Readme.md:1]: @MICHAEL-MAURICE can you please update your branch and remove this connector from your pull request?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
@MICHAEL-MAURICE may I ask you please to sign our CLA. you can do so by commenting the below command
```
@microsoft-github-policy-service agree company="Your Company"
```
Thank you so much again.

**Comment by MICHAEL-MAURICE** [comment]
@microsoft-github-policy-service agree company="1Me"

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@MICHAEL-MAURICE thank you so much for fixing your connector artifacts. 
Unfortunately you are still making changes to other connector in the repository. 
image
I would suggest you to close this PR and then update your branch.
image
Then create a new PR with your connector files only. 
Thank you so much again for your hard work with us.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+          "x-ms-summary": "Extention"
+        },
+        "Status": {
+          "description": "the Status of The Invitation.",
+          "type": "string",
+          "x-ms-summary": "Status"
+        }
+      }
+    },
+    "x-ms-connector-metadata": [
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:328]: @MICHAEL-MAURICE this property should be within the main root. please move them outside of the definition section.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://www.1me.live/privacy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "IT Operations;Website"
+      }
+    ],
+    "securityDefinitions": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:342]: @MICHAEL-MAURICE this property should be within the main root. please move them outside of the definition section.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+      "api_key": {
+        "type": "apiKey",
+        "in": "header",
+        "name": "XApiKey"
+      }
+    },
+    "security": [],
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:349]: @MICHAEL-MAURICE this property should be within the main root. please move them outside of the definition section.

File: certified-connectors/1Me Corporate/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,352 @@
...
+    "tags": []
```
Comment (Amjed-Ayoub) [inline certified-connectors/1Me Corporate/apiDefinition.swagger.json:350]: @MICHAEL-MAURICE this property should be within the main root. please move them outside of the definition section.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@MICHAEL-MAURICE 
Unfortunately you are still making changes to other connector in the repository.
I would suggest you to close this PR and then update your branch.
image
Then create a new PR with your connector files only.
Thank you so much again for your hard work with us.

**Pull request closed without merging**